docker-compose.yml

```diff
@@ -9,17 +8,23 @@ services:
         volumes:
             - postgres12:/var/lib/postgresql/data
         restart: on-failure
+
+        # If we want to connect to the server from outside
+        ports:
+          - 5432:5432
+
     mauro-data-mapper:
         image: "maurodatamapper/mauro-data-mapper:${MDM_TAG}"
         build:
             context: mauro-data-mapper
             args:
                 MDM_APPLICATION_COMMIT: "${MDM_APPLICATION_COMMIT}"
                 MDM_UI_COMMIT: "${MDM_UI_COMMIT}"
-                ADDITIONAL_PLUGINS: ""
-                MDM_UI_THEME_NAME: "default"
+                ADDITIONAL_PLUGINS: "uk.ac.ox.softeng.maurodatamapper.plugins:mdm-plugin-nhs-data-dictionary:2.0.0-SNAPSHOT"
+                MDM_UI_THEME_NAME: "nhs-digital"
                 CACHE_BURST: "${CACHE_BURST}"
                 MDM_EXPLORER_API_ENDPOINT: "http://localhost:${MDM_PORT}/api"
```

Comment:
We could probably remove references to the MDM Explorer, I don't think NHSE will be using it.

Reply:
The explorer UI has been replaced with the orchestration UI